assets/css/color.css

```diff
@@ -15,3 +21,51 @@ a {
   font-size: 20px;
   line-height: 32px;
 }
+
+input:invalid {
+  border: 1px solid red;
+}
+
+input:valid {
+  border: 1px solid darkgreen;
+}
+
+.mt-0 {
+  margin-top: 0 !important;
+}
+
+.mb-0 {
+  margin-bottom: 0 !important;
+}
+
+.pb-0 {
+  padding-bottom: 0 !important;
+}
+
+#intro {
+  background: url("https://dev-mamun.github.io/microverse-portfolio/assets/img/header-illsutration-mobile.svg") no-repeat;
+  background-size: cover;
+  background-position: center;
+}
```

Comment:
- [ ] **As suggested by the previous reviewer,** I think it would be better to use different background images for mobile and desktop versions to replicate the Figma design 😎😎

    Hint: You may try to use the following 👇 on desktop version
    ![Header-llustration-desktop](https://user-images.githubusercontent.com/102757126/214843140-0ccf3e80-3beb-4f4a-b678-510f4781798e.png)